src/couch_file.erl

```diff
@@ -16,13 +16,18 @@
 
 -include_lib("couch/include/couch_db.hrl").
 
-
 -define(INITIAL_WAIT, 60000).
 -define(MONITOR_CHECK, 10000).
 -define(SIZE_BLOCK, 16#1000). % 4 KiB
 -define(READ_AHEAD, 2 * ?SIZE_BLOCK).
 -define(IS_OLD_STATE(S), tuple_size(S) /= tuple_size(#file{})).
+-define(PREFIX_SIZE, 5).
+-define(DEFAULT_READ_COUNT, 1024).
 
+-type block_id() :: non_neg_integer().
+-type location() :: non_neg_integer().
+-type data() :: string() | binary() | eof.
+-type header_size() :: non_neg_integer().
```

Comment:
I'm personally not a fan of specs for module private functions but I won't go so far as to say to remove them.